Proposed fix for GTK 2+ Makefile.am reference to locale directory.
"$(DATADIRNAME)" is not defined in newest autotools versions, and
"$(localedir)" is preferred.

Signed-off-by: David McIlwraith <archaios@archaios.net>
---
 Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: gtk-v2/src/Makefile.am
===================================================================
--- gtk-v2/src/Makefile.am	(revision 15791)
+++ gtk-v2/src/Makefile.am	(working copy)
@@ -21,7 +21,7 @@
 
 INCLUDES = \
 	-DPACKAGE_DATA_DIR=\""$(datadir)"\" \
-	-DPACKAGE_LOCALE_DIR=\""$(prefix)/$(DATADIRNAME)/locale"\" \
+	-DPACKAGE_LOCALE_DIR=\""$(localedir)"\" \
 	-I$(top_builddir)/common \
 	-I$(top_srcdir)/common \
 	-I$(top_srcdir)/common/shared \
